### PR TITLE
Feature/event routing

### PR DESCRIPTION
**Pull request by srhboo**
note: storing 'current event' in state on router change has not been done in this PR

### CODE REVIEW

File: src/components/EventDetails.test.js [hunk 1]
```diff
@@ -3,29 +3,30 @@ import ReactDOM from 'react-dom';
...
-      url: "https://img.evbuc.com/https%3A%2F%2Fcdn.evbuc.com%2Fimages%2F28253674%2F183837305730%2F1%2Foriginal.jpg?h=200&w=450&rect=0%2C0%2C1000%2C500&s=b42f77f4a43a35ab28df4b3276da3494",
-    }
-  };
+  const props = {
+    event: {
+      name: {
+        text: "Toronto Code & Coffee | Third Sunday of Each Month in 2017",
+      },
+      url: "https://www.eventbrite.ca/e/toronto-code-coffee-third-sunday-of-each-month-in-2017-tickets-30433133321",
+      description: {
```
Comment (turquoisemelon) [inline src/components/EventDetails.test.js]: There seems to be some extra spacing in here. Just for the sake of formatting, is it possible to rid of it?

File: src/components/EventDetails.js [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
 
-export const EventDetails = (event) => {
+export const EventDetails = (props) => {
   return(
     <div className='event-details-card'>
       <div className="event-details-container">
-        <h3>{event.name.text}</h3>
-        <img alt="Event" src={event.logo.url} />
+        <h3>{props.event.name.text}</h3>
+        <img alt="Event" src={props.event.logo.url} />
```
Comment (turquoisemelon) [inline src/components/EventDetails.js]: Is it necessary to add a check like ```props.event.logo ?  : null``` in here? I'm not sure either but wanted to point it out. While I was trying to get event images inside the eventsByCategoryPage component, I was getting error without this check

**Comment by turquoisemelon** [comment]
Left a couple of comments. I didn't see any major issue otherwise. Good job 👍

File: src/components/EventDetails.js (hunk 2 above)
Comment (epq) [inline src/components/EventDetails.js]: Yes, you're right. I think we should add a check to see if we have a logo before displaying it. I overlooked that when I made the `EventDetails` component. According to the docs, the logo is optional: https://www.eventbrite.ca/developer/v3/response_formats/event/#ebapi-std:format-event

File: src/components/EventDetails.test.js (hunk 1 above)
Comment (cgolinski) [inline src/components/EventDetails.test.js]: I fetched the branch, and there doesn't seem to be any extra space here. Could GitHub's representation of the added and removed lines be confusing you? 

That said, there is one blank line that might be unnecessary: line 31, just after the end of the it().

File: src/constants/EventShellTemp.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export const eventShellTemporary = {
+    name: "fun event",
```
Comment (cgolinski) [inline src/constants/EventShellTemp.js]: I know this eventShell is just a placeholder, but it you might want to update it to: `name: {text: "fun event"};`. Name is not currently rendering.

**Comment by cgolinski** [comment]
Looks great! Left a couple minor comments.

File: src/components/EventDetails.js (hunk 2 above)
Comment (srhboo) [inline src/components/EventDetails.js]: good point - adding

File: src/constants/EventShellTemp.js (hunk 3 above)
Comment (srhboo) [inline src/constants/EventShellTemp.js]: good catch!

File: src/components/EventDetails.test.js (hunk 1 above)
Comment (srhboo) [inline src/components/EventDetails.test.js]: deleted line 31

**Pull request merged**